src/build_artifact.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+ARTIFACT_FILE_NAME="${BUILD_FILE_NAME:-lambda-package.zip}"
+
+output_directory="${LAMBDA_TASK_ROOT}/output"
+artifact_path="$output_directory/$ARTIFACT_FILE_NAME"
+
+# Build the deployment package by zipping the libraries and handler. We
+# explicitly exclude the build/ and output/ subdirectories as they are
+# not part of the deployment package.
+cd "${LAMBDA_TASK_ROOT}" && zip -rq9 "$artifact_path" . -x "build/*" -x "output/*"
```

Comment:
Prefer long flag names:
```suggestion
cd "${LAMBDA_TASK_ROOT}" && zip --recurse-paths --quiet -9 "$artifact_path" . --exclude "build/*" --exclude "output/*"
```

Reply:
Ah, I didn't realize those had long flag names (curse you `zip -h` output!). Please see [72c616e](https://github.com/cisagov/skeleton-aws-lambda-python/pull/3/commits/72c616ebdbe0586c0911de98b647778cc832a11a) for implementation.